pkgs/data/misc/ipfire-geoip/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, tor
+, python3
+, mmdbctl
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "ipfire-geoip";
+  inherit (tor.geoip) version;
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ python3 mmdbctl ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    python3 ${./csv-convertor.py} ${tor.geoip}/share/tor/geoip geoip.csv
```

Comment:
Or shall we directly download the database from https://location.ipfire.org/databases/1/archive/

Reply:
We can download it from https://github.com/torproject/tor/tree/main/src/config. If we download the original database we need to package https://github.com/torproject/tor/tree/main/scripts/maint/geoip and location to convert it from .db to .txt to the Tor csv.